src/main/java/uk/gov/ons/census/casesvc/service/BlankQuestionnaireService.java

```diff
@@ -0,0 +1,178 @@
+package uk.gov.ons.census.casesvc.service;
+
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.CONT_FORM_TYPE;
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.HH_FORM_TYPE;
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.IND_FORM_TYPE;
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.mapQuestionnaireTypeToFormType;
+import static uk.gov.ons.census.casesvc.utility.MetadataHelper.buildMetadata;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.casesvc.model.dto.ActionInstructionType;
+import uk.gov.ons.census.casesvc.model.dto.EventTypeDTO;
+import uk.gov.ons.census.casesvc.model.dto.Metadata;
+import uk.gov.ons.census.casesvc.model.entity.Case;
+import uk.gov.ons.census.casesvc.model.entity.UacQidLink;
+
+@Component
+public class BlankQuestionnaireService {
+
+  private final CaseService caseService;
+
+  private Map<BlankQuestionnaireService.Key, BlankQreRule> rules = new HashMap<>();
```

Comment:
This could be static and initialised in a static block, the same as other rule tables have been implemented

Reply:
Can do but this was copied from what was done for the receipting rules. I'd want to change that as well so they use the same pattern.

Reply:
Actually scratch that, it can't easily be made static because the rule class/functions are not.